dozer-types/src/arrow_types/to_arrow.rs

```diff
@@ -152,10 +152,7 @@ pub fn map_field_type(typ: FieldType, metadata: Option<&mut HashMap<String, Stri
             metadata.map(|m| m.insert("logical_type".to_string(), "Binary".to_string()));
             DataType::Binary
         }
-        FieldType::Bson => {
-            metadata.map(|m| m.insert("logical_type".to_string(), "Bson".to_string()));
-            DataType::Binary
-        }
+        FieldType::Json => todo!(),
```

Comment:
Maybe we serialize JSON to arrow String and add a logical type? @v3g42 what do you think?

Reply:
Yup makes sense.

Reply:
so then to_arrow is JSON -> String but how are we handling the String -> Json in from arrow?, how to differentiate with the Normal String?

and @chubei last time you told me to return error for json what should we do?

Reply:
Let's go with converting to String. I believe it's better than returning an error.

When converting to arrow, we insert metadata saying the `logical_type` is `json`, and when converting from arrow, we check if that metadata is present and convert accordingly.

But let's do this in the next PR. In this PR we can just convert JSON to string without converting back.